docs/README.md

```diff
@@ -85,13 +85,13 @@ Please refer to this [page](objects.md) for details on how AKO interprets the Ku
 Please refer to this [page](cc_to_ako.md) for details on how to migrate workloads from cloud connector based Avi controller to AKO based Avi controller.
 
 ### AKO Compatibility Guide
-AKO version 1.9.1 support for Kubernetes, Openshift, Avi Controller is as below:
+AKO version 1.10.1 support for Kubernetes, Openshift, Avi Controller is as below:
 
 | **Orchestrator/ Controller** | **Versions Supported** |
 | --------- | ----------- |
-| `Kubernetes` | 1.22 - 1.25 |
-| `Openshift` | 4.6 - 4.11 |
-| `Avi Controller` | 21.1.5 - 22.1.3 |
+| `Kubernetes` | 1.22 - 1.26 |
+| `Openshift` | 4.6 - 4.12 |
```

Comment:
@akshayhavile is it required to support 6 versions of openshift and 4 versions of k8s?

Reply:
@swathinsankaran at least for the ako operator, we are supporting from Openshift 4.8.